src/runtime/plan/README.md

```diff
@@ -0,0 +1,83 @@
+# Planning in the Shell
+
+By default plans are produced and consumed locally in the browser.
+When using [arcs-live](https://polymerlabs.github.io/arcs-live/shells/web-shell) by default both arcs and suggestions are serialized to Firebase.
+
+## URL params
+Suggestions storage can be controlled by **plannerStorage** url param. 
+
+To use volatile storage for suggestion go to:  
+https://polymerlabs.github.io/arcs-live/shells/web-shell?plannerStorage=volatile
+
+To use PouchDb storage for suggestion go to:  
+https://polymerlabs.github.io/arcs-live/shells/web-shell?plannerStorage=pouchdb://local/random-db-name  
+The suggestion values will be found in Chrome's inspector, under Application > Storage > Local Storage.
+
+
+To populate Arcs Extension strategy explorer, add the **plannerDebug** url param:  
+https://polymerlabs.github.io/arcs-live/shells/web-shell?plannerDebug
+
+# Remote Planning 
+To consume plans produced on the server, disable shell planning with an **onlyConsumer** url param, ie:  
+https://polymerlabs.github.io/arcs-live/shells/web-shell?plannerOnlyConsumer=true
+
+## Firebase
+To run planning on the server use the following command:
+
+```
+/shells/planner-shell $ serve.sh
+```
+or debug with:
+```
+/shells/planner-shell $ debug.sh
+```
+
+The remote planning is performed per user for all their arcs.  
+The default user is **planner** (currently to override, set environment variable or update code [link](https://github.com/PolymerLabs/arcs/blob/master/shells/planner-shell/index.js#L23)).
+
+## PouchDB
+To test remote planning with Pouch you will need to run a Pouch Server instance by following these steps:
+
+1. Build the server by executing:
+```
+/server $ npm run server
+```
+2. Start the server on port 8080 by running 
+```
+/server $ env {args} npm run server:start
+```
+Optionally prefix your command with the following optional environment arguments:
+```
+DEBUG=true
+ARCS_USER_ID=xxxxx
+STORAGE_KEY_BASE  (default is pouchdb://localhost:8080/user/)
+```
+Navigate to the Arcs page and use the storage, **plannerStorage** and **plannerOnlyConsumer** parameters:  
+https://localhost:8080/shells/web-shell?onlyConsumer=true&storage=pouchdb://localhost:8080/user/&plannerStorage=pouchdb://localhost:8080/user/
+
+
```

Comment:
Add a TODO to add instructions on how to bring up a Pouch server on a Personal Cloud server once we've gotten far enough to do so.

Reply:
...by which I mean, we'll probably link to them, not write them all out here. Perhaps linking to `server/README.md` is sufficient for now.

Reply:
added a link to server/readme